ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelContainer/v1/panelContainer/new/.content.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any specific reason to put it under `new` directory?

Reply:
something which we inherit from sites container